.env.test_perm

```
@@ -0,0 +1 @@
+SECRET=updated
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)